Makefile

```diff
@@ -38,7 +38,7 @@ ifeq ($(LEDGER_ENABLED),true)
 endif
 
 ifeq (cleveldb,$(findstring cleveldb,$(GAIA_BUILD_OPTIONS)))
-  build_tags += gcc
+  build_tags += gcc cleveldb
```

Comment:
You sure this works? @marbar3778 
I suspect we should also add
```
CGO_ENABLED=1
```

Reply:
We have this on tendermint because if you don't use cleveldb we explicitly disable cgo. I believe cgo is enabled by default, (needed for ledger)